gnome/tracker/Portfile

```diff
@@ -123,6 +123,8 @@ configure.args      --disable-schemas-compile \
                     --enable-icon \
                     --enable-playlist
 
+configure.cflags-append -Wno-int-conversion
```

Comment:
is there an upstream issue that tracks the problem and/or do they have a solution lined up already (which presumably will be different than this)?

Reply:
AFAIU, Tracker 1.x has not been supported in many, many years.  When I get the chance, I want to try to update to 2.x so I can update Nautilus to a semi-modern version, but in the mean time, this is a simple stop-gap.